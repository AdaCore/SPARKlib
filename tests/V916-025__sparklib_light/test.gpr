project Test extends "sparklib_light_internal" is

   for Main use ("main.adb");
   for Runtime ("ada") use "light";

end Test;
