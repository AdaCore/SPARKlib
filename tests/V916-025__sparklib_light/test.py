from test_support import prove_all
from subprocess import call

prove_all(steps=600, opt=["-U"])
call(["gprbuild", "-q", "-P", "test.gpr"])
