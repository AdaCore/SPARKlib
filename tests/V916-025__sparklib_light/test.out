main.adb:27:13: info: implicit aspect Always_Terminates on "Hash" has been proved, subprogram will terminate
spark-big_reals__light.ads:34:32: warning: function Is_Valid is assumed to return True
